sellable/src/sellable_module.rs

```diff
@@ -0,0 +1,135 @@
+use std::{cell::RefCell, rc::Rc};
```

Comment:
This trait is to remove the duplication in code.
It creates a trait of similar functions both the Sellable and RSellable structs impl